planner/core/integration_test.go

```diff
@@ -3375,6 +3375,10 @@ func TestScalarFunctionPushDown(t *testing.T) {
 	tk.MustQuery("explain analyze select /*+read_from_storage(tikv[t])*/ * from t where ascii(e);").
 		CheckAt([]int{0, 3, 6}, rows)
 
+	rows[1][2] = "eq(json_valid(test.t.c), 1)"
+	tk.MustQuery("explain analyze select /*+read_from_storage(tikv[t])*/ * from t where json_valid(c)=1;").
+		CheckAt([]int{0, 3, 6}, rows)
+		
```

Comment:
It seems that this is not an empty line, but contains tab/space. Please delete them (or simply format the codes).

Reply:
I change this empty line